waves/utilities/validators.py

```diff
@@ -0,0 +1,38 @@
+"""Utility functions to validating model inputs for each of the models that can be called by
+WAVES.
+"""
+
+from typing import Any
+
+
+def check_dict_consistent(dict_values: dict, output_name: str) -> Any:
```

Comment:
I would slightly update the name to `check_dict_consistency` so it reads more like a command/sentence.